providers/fips/fipsprov.c

```diff
@@ -457,6 +461,8 @@ static const OSSL_ALGORITHM fips_signature[] = {
 #endif
     { "RSA:rsaEncryption", "provider=fips,fips=yes", rsa_signature_functions },
 #ifndef OPENSSL_NO_EC
+    { "ED25519", "provider=fips,fips=no", ed25519_signature_functions },
+    { "ED448", "provider=fips,fips=no", ed448_signature_functions },
```

Comment:
The "fips=no" here makes sense - but might be unexpected for some.  We should document this somewhere.

Reply:
It will be mentioned in the security policy and on the FIPS certificate.  There is a non-approved algorithm section.

I've added a NEWS entry noting this. It seems more appropriate than a CHANGES entry.

Reply:
I this this section in provider.pod at least should be updated:

> The FIPS provider is a dynamically loadable module, and must therefore be loaded explicitly, either in code or through OpenSSL configuration (see config(5)). Should it be needed (if other providers are loaded and offer implementations of the same algorithms), the property "provider=fips" can be used as a search criterion for these implementations. All algorithm implementations in the FIPS provider can also be selected with the property "fips=yes".

Reply:
Done.